src/Surfnet/StepupMiddleware/CommandHandlingBundle/Tests/Configuration/CommandHandler/InstitutionConfigurationCommandHandlerTest.php

```diff
@@ -220,6 +251,98 @@ public function show_raa_contact_information_option_is_changed_if_its_given_valu
      * @test
      * @group command-handler
      */
+    public function allowed_second_factor_list_is_changed_if_its_values_are_different_than_the_current_list()
+    {
+        $institution                     = new Institution('Institution');
+        $institutionConfigurationId      = InstitutionConfigurationId::normalizedFrom($institution);
+        $useRaLocationsOption            = UseRaLocationsOption::getDefault();
+        $showRaaContactInformationOption = ShowRaaContactInformationOption::getDefault();
+        $originalAllowedSecondFactorList = AllowedSecondFactorList::blank();
+
+        $secondFactorsToAllow = ['sms', 'yubikey'];
+
+        $allowedSecondFactors = array_map(function ($secondFactorType) {
+            return new SecondFactorType($secondFactorType);
+        }, $secondFactorsToAllow);
+        $allowedSecondFactorList = AllowedSecondFactorList::ofTypes($allowedSecondFactors);
```

Comment:
Suggestion: remove the mapping in favor of simple instantiation:
```
$originalAllowedSecondFactorList = AllowedSecondFactorList::blank();
$updatedAllowedSecondFactorList  = AllowedSecondFactorList::ofTypes([new SecondFactorType('sms'), new SecondFactorType('yubikey')]);
```

Visually identifies the difference, less code that requires thinking about what it does and thus detracts from what we're actually doing here (testing that it all works).